src/components/screens/edit-surveys-screen.tsx

```diff
@@ -80,12 +80,17 @@ const EditSurveysScreen = () => {
       <Stack direction="row" flex={1}>
         <Editor setPanelProperties={ setPanelProperties } surveyId={survey.id} />
         <PropertiesPanel>
-          { panelProperties === EditorPanelProperties.SURVEY
+          { panelProperties.panelType === EditorPanel.SURVEY
             ? <SurveyProperties
                 survey={ survey }
                 onSaveSurvey={ onSaveSurvey }
               />
-            : <PageProperties /> }
+            : panelProperties.pageNumber &&
```

Comment:
Just a sidenote, there is no backend validation for pageNumbers, so this will cause problems if the pageNumber would ever be 0